.mergify.yml

```diff
@@ -127,36 +99,10 @@ pull_request_rules:
         - closed
       - and:
         - label=automation
-        - head~=^update-.*-version
+        - head~=^updatecli.*
```

Comment:
https://github.com/elastic/fleet-server/blob/main/.ci/bump-elastic-stack-snapshot.yml and https://github.com/elastic/fleet-server/blob/main/.ci/bump-golang.yml are the ones creating branches in the upstream with the name `updatecli`, for instance https://github.com/elastic/fleet-server/pull/2408